src/CMakeLists.txt

```
@@ -238,6 +238,10 @@ endif()
 find_package(Threads REQUIRED)
 target_link_libraries(rdkafka PUBLIC Threads::Threads)
 
+if(WITH_CURL)
+    target_link_libraries(rdkafka PUBLIC curl)
```

Since [FindCURL ](https://cmake.org/cmake/help/v3.0/module/FindCURL.html) is used to determine whether to link against it, the provided variables should be used:
```suggestion
    target_include_directories(rdkafka PUBLIC ${CURL_INCLUDE_DIRS})
    target_link_libraries(rdkafka PUBLIC ${CURL_LIBRARIES})
```